samples/client/petstore/python/tests/test_pet_api.py

```diff
@@ -157,7 +157,7 @@ def test_async_with_result(self):
         response = thread.get()
         response2 = thread2.get()
 
-        self.assertEquals(response.id, self.pet.id)
```

Comment:
assertEquals is depricated. pytest warned us about it
Deprecation info: https://docs.python.org/3.3/library/unittest.html#deprecated-aliases